src/pages/index.tsx

```diff
@@ -14,6 +14,12 @@ const genericOverview = [
     imageUrl: 'img/undraw_docusaurus_mountain.svg',
     summary: <>リファレンス</>,
   },
+  {
+    title: 'Distribution Guide',
+    to: 'distribution',
+    imageUrl: 'img/undraw_docusaurus_mountain.svg',
+    summary: <>リファレンス</>,
```

Comment:
こんな感じ？トップに配置するかどうか、少し悩ましい感じもあるけど、どうだろう。
```suggestion
    summary: <>アプリケーションの配布</>,
```

Reply:
トップへの配置はやめた